config.json

```
@@ -9,7 +9,8 @@
   "addArticleLinkInContent": true,
   "allowImages": true,
   "outputFileName": "output.epub",
-  "maxArticleCount": 100,
+  "maxArticleCount": 15,
+  "searchQuery": "sort:saved-asc",
```

I would prefer to keep the default article order as descending with ascending order being an option documented in readme
```suggestion
  "searchQuery": "sort:saved-desc",
```